keyboards/svalboard/trackball/pmw3389/rules.mk

```diff
@@ -0,0 +1,19 @@
+# this turns on trackballs including pimoroni
+POINTING_DEVICE_ENABLE = yes
+# this selects trackball driver - moved to left/right for 3360 default, and 3389 subfolders for 3389
+POINTING_DEVICE_DRIVER = pmw3389   
+# pimoroni_trackball
+# pmw3389
+MH_AUTO_BUTTONS = yes
+
+ifeq ($(strip $(POINTING_DEVICE_DRIVER)), pimoroni_trackball)
```

Comment:
We should remove this stuff.  pimoroni / the other sensor, from both rules.mk.

Minor cleanup.  If urgent, merge, and can be addressed later.